src/metric/definition.go

```diff
@@ -718,6 +718,9 @@ var KSMSpecs = definition.SpecGroups{
 	"node": {
 		TypeGenerator: kubeletMetric.FromRawGroupsEntityTypeGenerator,
 		Specs: []definition.Spec{
+			// nodeName and Labels should be provided here as well so conditions can be queried and faceted by them.
+			{Name: "nodeName", ValueFunc: prometheus.FromLabelValue("kube_node_info", "node"), Type: sdkMetric.ATTRIBUTE},
+			{Name: "label.*", ValueFunc: prometheus.InheritAllLabelsFrom("node", "kube_node_labels"), Type: sdkMetric.ATTRIBUTE},
```

Comment:
Just noticed that these two funcs have their arguments in opposite positions and now I won't be able to sleep at night.